ArduCopter/Parameters.cpp

```diff
@@ -1208,6 +1208,13 @@ const AP_Param::GroupInfo ParametersG2::var_info2[] = {
     // @User: Advanced
     AP_GROUPINFO("SURFTRAK_TC", 5, ParametersG2, surftrak_tc, 1.0),
 
+    // @Param: TKOFF_TH_MAX
+    // @DisplayName: Maximum throttle during take-off ramp up
+    // @Description: Maximum throttle allowed before controllers assume the aircraft is airborn druing the takeoff process.
```

Comment:
```suggestion
    // @Description: Maximum throttle allowed before controllers assume the aircraft is airborne during the takeoff process.
```

Reply:
I think this could be very usefull but I don't have anything to link to at this time.